src/main/java/mil/dds/anet/config/AnetConfiguration.java

```diff
@@ -138,6 +166,15 @@ public void setServerUrl(String serverUrl) {
     this.serverUrl = serverUrl;
   }
 
+  public String getAnetDictionaryName() {
+    return anetDictionaryName;
+  }
+
+  public void setAnetDictionaryName(String anetDictionaryName) throws Exception {
+    this.anetDictionaryName = anetDictionaryName;
+    this.loadDictionary();
+  }
```

Comment:
Maybe add method Javadoc clearly stating that setting this field will incur a side-effect: the file will be loaded (or at least attempted to load). And make the thrown exceptions somewhat more specific than just `Exception` (and document them in the Javadoc while you're at it).